packages/lesmis/src/api/mutations/useLogin.js

```diff
@@ -21,6 +21,8 @@ export const useLogin = () => {
     {
       onSuccess: () => {
         queryClient.invalidateQueries('user');
+        queryClient.invalidateQueries('entity');
+        queryClient.invalidateQueries('entities');
       },
```

Comment:
Clear the cached data for these endpoints after login so that the data relating to the new permissions is picked up.